client/src/components/createcontact3.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
```

Comment:
would also loe to know what use navigate is and how it works :)

Reply:
Yes, let's connect it is what helped me set up my routers.